.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
+
+[*.sh]
+indent_size = 2
+indent_style = tab
```

Comment:
isn't it better to use space indents for scripts?

Reply:
No scripts should be always be TABS

bash has for example EOF (HEREDOC) that reacts different on TABs vs SPACES
more details:
![image](https://github.com/user-attachments/assets/e071e8c3-7e99-47e8-87b2-2b15fb62a511)